archeo/theme.json

```diff
@@ -231,7 +231,7 @@
 			},
 			"core/separator": {
 				"color": {
-					"background": "var(--wp--preset--color--foreground)"
+					"text": "var(--wp--preset--color--foreground)"
```

Comment:
I don't believe this should change to 'text'.  I believe 'background' is still the correct attribute to style.